File: algorithm/LeetCode/MissingNumber/MissingNumber.java
public class Solution {
    public int missingNumber(int[] nums) {
        int sum = nums.len * (nums.len + 1)/2; //length大小是n-1
        for(int i = 0; i < nums.length; ++i){
            sum -= nums[i];
        }
        return sum;
    }
}

File: algorithm/LeetCode/MissingNumber/MissingNumber.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int sz = nums.size();
        int total = (sz + 1)*sz / 2;
        for (int i = 0; i < sz; ++i){
            total -= nums[i];
        }
        return total;
    }
};